httpobs/scanner/analyzer/content.py

```diff
@@ -101,6 +101,8 @@ def subresource_integrity(reqs: dict, expectation='sri-implemented-and-external-
         sri-implemented-but-external-scripts-not-loaded-securely: SRI implemented, but with scripts loaded over HTTP
         sri-not-implemented-but-external-scripts-loaded-securely: SRI isn't implemented,
           but all scripts are loaded over HTTPS
+        sri-not-implemented-and-same-origin-scripts-not-loaded-securely: SRI isn't implemented,
+          and same-origin scripts are not downloaded over HTTPS
```

Comment:
Rather than have a new penalty, could we instead just roll this into `sri-not-implemented-and-external-scripts-not-loaded-securely` and update its description?